src/utils/LiveAnnouncer.tsx

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import React, { Fragment, useImperativeHandle, useRef, useState } from "react";
+import ReactDOM from "react-dom";
+import { VisuallyHidden } from "@react-aria/visually-hidden";
+
+/* Inspired by https://github.com/AlmeroSteyn/react-aria-live */
+const liveRegionAnnouncer = React.createRef();
+let node: any = null;
+let clearTimeoutId: any = null;
+const LIVEREGION_TIMEOUT_DELAY = 1000;
+
+type TAriaLive = "assertive" | "off" | "polite" | undefined;
+
+/**
+ * Announces the message using screen reader technology.
+ */
+export function announce(
```

Comment:
Could not seem to do `yarn add @react-aria/live-announcer` so i had to copy paste this here.